styles/index.css

```diff
@@ -0,0 +1,189 @@
+div {
+  box-sizing: border-box;
+  text-align: center;
+}
+
+.page-header div,
+.images-wrapper,
+.blocks-num-four div,
+.blocks-num-five,
+nav,
+footer {
+  border: 1px solid #ccc;
+}
+
+.page-header {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  grid-gap: 5px;
+  margin: 5px 0;
+}
+@media (min-width: 360px) {
+  .page-header {
+    grid-template-columns: 1fr 4fr;
+  }
+}
+@media (min-width: 1024px) {
+  .page-header {
+    grid-template-columns: 1fr 7fr;
+  }
+}
+.page-header div {
+  padding: 0.5em;
+}
+.page-header div:first-child {
+  background-color: #216583;
+}
+.page-header div:last-child {
```

Comment:
here and below: use **.class** as a selector

Reply:
Done.
Thanks for the review.